docs/python/testing.md

```diff
@@ -234,6 +234,21 @@ Support for running tests in parallel with pytest is available through the `pyte
 
 3. Run your tests, which will now be run in parallel.
 
+## Run tests with coverage
+
+Test coverage is a measure of how much of your code is covered by your tests, which can help you identify areas of your code that are not being fully tested. For more information on test coverage, visit VS Code's [Test Coverage documentation](/docs/editor/testing#test-coverage.md).
+
+To run tests with coverage, select the coverage run icon in the Test Explorer or “Run with coverage” from any menu you normally trigger test runs from. The Python extension will run coverage using the [`pytest-cov`](https://pypi.org/project/pytest-cov/) plugin if you are using pytest, or with [`coverage.py`](http://coverage.py/) for unittest.
```

Comment:
```suggestion
To run tests with coverage enabled, select the coverage run icon in the Test Explorer or the “Run with coverage” option from any menu you normally trigger test runs from. The Python extension will run coverage using the [`pytest-cov`](https://pypi.org/project/pytest-cov/) plugin if you are using pytest, or with [`coverage.py`](https://coverage.readthedocs.io/) for unittest.
```

Reply:
also not confident on the coverage.py link, just know that the previous one didn't work for me